documentation/source/_templates/layout.html

```diff
@@ -0,0 +1,12 @@
+{% extends "!layout.html" %}
+
+{% block content %}
+
+<div style="background-color: rgb(255, 248, 225); color: #444; padding: 1em; border: 1px solid rgb(255, 111, 0); margin-top: 5px; margin-bottom: 5px;">
+  The <a href="https://docs.google.com/forms/d/e/1FAIpQLSfD36PldzszbuZjysss3AMvxkf6XCtSbDTVh9qVNNYDaHTZ_w/viewform">cocotb user survey 2020</a> is awaiting your answers.
+  Please take 10 minutes to <b>help cocotb developers understand your use of cocotb better, and influence the direction of the project.</b>
+  <b><a href="https://docs.google.com/forms/d/e/1FAIpQLSfD36PldzszbuZjysss3AMvxkf6XCtSbDTVh9qVNNYDaHTZ_w/viewform">Click here to take the survey now!</a></b>
+</div>
```

Comment:
After putting this in `content` as I suggest above, you can piggy back on RST styles
```suggestion
<div class="admonition primary" style="display: table">
  <p>The cocotb user survey 2020 is awaiting your answers.
  Please take 10 minutes to help cocotb developers understand your use of cocotb better, and influence the direction of the project.</p>
  <a href="https://docs.google.com/forms/d/e/1FAIpQLSfD36PldzszbuZjysss3AMvxkf6XCtSbDTVh9qVNNYDaHTZ_w/viewform" class="btn btn-primary float-right">Take the survey now!</a>
</div>
```

Reply:
![image](https://user-images.githubusercontent.com/425260/78223467-d2f37400-74be-11ea-8619-5a3cba2cd61e.png)

Reply:
(feel free to add back the bold as needed)

Reply:
Or:
```suggestion
<div class="admonition info" style="display: table">
  <p class="admonition-title">The cocotb user survey 2020</p>
<p>Please take 10 minutes to help cocotb developers understand your use of cocotb better, and influence the direction of the project.</p>

  <a href="https://docs.google.com/forms/d/e/1FAIpQLSfD36PldzszbuZjysss3AMvxkf6XCtSbDTVh9qVNNYDaHTZ_w/viewform" class="btn btn-info float-right">Take the survey now!</a>
</div>
```
![image](https://user-images.githubusercontent.com/425260/78229881-7bf29c80-74c8-11ea-94e2-f87f6af31812.png)